chainer/functions/normalization/weight_standardization.py

```diff
@@ -0,0 +1,101 @@
+from chainer import cuda
+from chainer import function_node
+import chainer.functions
+from chainer.utils import type_check
+import six
+
+
+def _broadcast_to(xp, x, shape):
+    # xp: numpy, cupy, or chainer.functions
+    if hasattr(xp, 'broadcast_to'):
+        return xp.broadcast_to(x, shape)
+    else:
+        # numpy 1.9 doesn't support broadcast_to method
+        dummy = xp.empty(shape)
+        bx, _ = xp.broadcast_arrays(x, dummy)
+        return bx
+
+
+class WeightStandardization(function_node.FunctionNode):
+
+    """Weight standardization"""
+
+    def __init__(self, eps=1e-5):
+        self.eps = eps
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        x_type, = in_types
+
+        type_check.expect(
+            x_type.dtype.kind == 'f',
+            x_type.ndim >= 2,
+        )
+
+    def _compute(self, xp, x):
+        # xp: numpy, cupy, or chainer.functions
+        axes = tuple(six.moves.range(1, len(x.shape)))
+        mu = xp.mean(x, axis=axes, keepdims=True)
+        x_mu = x - _broadcast_to(xp, mu, x.shape)
```

Comment:
Automatic broadcast could be used (#4679).

Reply:
If using `chainer.functions.broadcast_to`, we will get `chainer.variable` instead of `cupy.ndarray`.
Do you suggest it is better to use `variable` isntead of `ndarray` in those computation?

Reply:
I meant `x_mu = x - mu` works.

Reply:
I'm sorry.  I should have commented to the lines in `backward`.

Reply:
Ah! I see, I fixed it.